### PR TITLE
update: built-in-functions.md

### PR DESCRIPTION
**Pull request by rrayhka**
### Description
I changed python's built-in function definitions and added usage examples

### Issue Solved
#5386 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/built-in-functions/built-in-functions.md [hunk 1]
```diff
@@ -13,6 +13,39 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The Python interpreter has a set of functions and types built into it (pre-defined). They are always ready at your disposal; you can use them without needing to import a library.
+**Built-in functions** in Python are a set of pre-defined functions that are readily available to use without the need to import any external libraries. These functions provide essential functionality for performing common tasks, such as manipulating data types, handling input/output, and more. Python includes 68 built-in functions, which are categorized and listed alphabetically for easy reference.
+
+
+
```
Comment (cigar-galaxy82) [inline content/python/concepts/built-in-functions/built-in-functions.md:19]: Only give 1 blank line after the para so remove 2 blank lines

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @rrayhka thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Comment by rrayhka** [comment]
> Hi @rrayhka thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

okay, wait a sec

**Comment by rrayhka** [comment]
@cigar-galaxy82  done man

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @rrayhka 😄 

The entry is ready to be merged! 🚀

**Pull request merged**